builder.go

```diff
@@ -544,3 +564,13 @@ func coalesce[T comparable](l ...T) T {
 
 	return c
 }
+
+// validateVarName checks whether the first rune is a valid starting character
+// for a variable name according to the specified criteria.
+func validateVarName(varName string) {
```

Comment:
Remove the validation - I'd rather have no validation than an incomplete one, it validates only the first character.

Two issues:
* panics, interrupts the build flow - we store error in the Builder struct
* incomplete validation.